compose/mpp/demo/src/uikitMain/kotlin/bugs/IosBugs.kt

```diff
@@ -24,5 +24,6 @@ val IosBugs = Screen.Selection(
     KeyboardEmptyWhiteSpace,
     KeyboardPasswordType,
     UIKitRenderSync,
-    DispatchersMainDelayCheck
+    DispatchersMainDelayCheck,
+    SelectionContainer,
```

Comment:
I guess we need to add such tests into common code. This list was made to call API from uikitMain source set, but now it's used not for this purpose

Reply:
Done!
Moved to commonMain "BugReproducers" section.
Also moved all bug reproducers to separate package.